### ISSUE TITLE
multicursor after region is selected

### ISSUE THREAD

**Issue by chaoky** [jyp/boon#86]
having @ available after selecting a region with SPC woulb be nice

**Comment by jyp** [comment]
I think that you mean that you want @ as a command to refine the current selection. Is this correct?

**Comment by chaoky** [comment]
yes, doing `spc  @ selection region` is fine but `selection @ region` seems quicker and more visual, I'll maybe try to implement this later today

**Comment by jyp** [comment]
I was expecting that you meant 'spc region @ selection' but that's problematic, because the 'selection' is not evident to make from having a current region. 'spc selection @ region' is not too difficult, but it's also not very useful ?
